bin/vyper

```diff
@@ -56,7 +56,7 @@ parser.add_argument(
 parser.add_argument(
     '--version',
     action='version',
-    version='{0}'.format(vyper.__version__),
+    version='{0}\nGIT:{1}'.format(vyper.__version__, vyper.__commit__),
```

Comment:
```suggestion
    version='{0}+commit.{1}'.format(vyper.__version__, vyper.__commit__),
```

Reply:
Do this so it's compatible with semvar

Reply:
Also, use the first 7 chars of the commit hash, like git.